app/templates/jinja2/index.html

```diff
@@ -98,12 +98,36 @@ <h1>Signups Schedule</h1>
             <td>10am-5pm</td>
             <td>10am-5pm</td>
           </tr>
+          <tr>
```

Comment:
Similar to the signup email dates/times, we should have a way to set a time/calendar widget of some sort so we don't need to manually edit this all the time. Again, if you could keep these changes on prod only and just leave them out of the PR while I get an issue written up for this, that'd be great.